src/hypermedia-constants.js

```diff
@@ -204,7 +204,8 @@ export const Rels = {
 	},
 	IANA: {
 		edit: 'edit',
-		preview: 'preview'
+		preview: 'preview',
```

Comment:
what `IANA` means ?

Reply:
looks like its some sort of standard authority? 
https://www.iana.org/assignments/link-relations/link-relations.xhtml